library/date/strftime_spec.rb

```diff
@@ -23,19 +23,9 @@
     @date.strftime("%Z").should == "+00:00"
   end
 
-  # %v is %e-%b-%Y for Date/DateTime
-  version_is date_version, ""..."3.2" do #ruby_version_is ""..."3.1" do
```

Comment:
Is it correct to remove this section guarded with `version_is date_version, ""..."3.2"`?

Reply:
Assuming the comment is correct (`ruby_version_is ""..."3.1" do`, so it should be behavior before Ruby 3.1) it should be.